src/main/java/teammates/common/datatransfer/questions/FeedbackResponseDetails.java

```diff
@@ -29,7 +29,11 @@ public abstract void extractResponseDetails(
 
     public abstract String getAnswerString();
 
-    public abstract String getAnswerHtml(FeedbackQuestionDetails questionDetails);
+    public abstract String getAnswerHtmlInstructorView(FeedbackQuestionDetails questionDetails);
+
+    public String getAnswerHtmlStudentView(FeedbackQuestionDetails questionDetails) {
+        return getAnswerHtmlInstructorView(questionDetails);
+    }
```

Comment:
Seems to be different from what @damithc suggested, but this version might still work (using instructor view as the default, which needs to be implemented, and overriding `getAnswerHtmlStudentView` if necessary)